src/Processors/Formats/Impl/ArrowBlockInputFormat.h

```diff
@@ -42,6 +47,8 @@ class ArrowBlockInputFormat : public IInputFormat
     const FormatSettings format_settings;
 
     void prepareReader();
+
+    std::atomic<int> is_stopped{0};
```

Comment:
Added this flag even though there is already `std::atomic<bool> is_cancelled` in the base class, because `copyData` accepts atomic from `int`, not from `bool`.